docs/glossary.rst

```diff
@@ -0,0 +1,83 @@
+Glossary
+========
+
+.. glossary::
+   :sorted:
+
+   WebMon
+      Monitor for instrument status and autoreduction status at SNS and HFIR. WebMon can refer to
+      either the whole system or the landing page at https://monitor.sns.gov.
+
+   DASMON
+      Data Acquisition System Monitor (DASMON) is a per-beamline service that reports PV values and
+      experiment/run meta-data to WebMon. Due to the high volume of PV updates, DASMON writes PV
+      updates straight to the database. DASMON reports its status to WebMon for system diagnostics.
+
+   DAS
+      Data Acquisition System.
+
+   SMS
+   Stream Management Service
+      Service that aggregates data from fast neutron event data and slow PVs into a data stream
+      available for both live monitoring and file archiving. SMS reports its status to WebMon for
+      system diagnostics.
+
+   STC
+   Streaming Translation Client
+      Service that translates the experiment data stream from SMS to NeXus data file. Triggers the
+      post-processing workflow after the NeXus file has been created.
+
+   PVSD
+   Process Variable Streaming Daemon
+      Per-beamline service that subscribes to EPICS Control System PV:s and forwards PV value
+      changes to the SMS. PVSD reports its status to WebMon for system diagnostics.
+
+   PV
+      Process Variable, for example sample environment variables.
```

Comment:
Process Variable (PV) are metadata about an instrument, run, or experiment.  Subsets of PVs are used for monitoring